src/requests/completions.jl

```diff
@@ -302,7 +292,7 @@ function path_completion(doc, offset, rng, t, CIs)
     if !startswith(path, "/")
         path = joinpath(_dirname(uri2filepath(doc._uri)), path)
     end
-    if _ispath(path)
+    try
```

Comment:
Does this need to be around the whole loop?

Reply:
I tweaked it a bit. I think the try catch around the whole loop should stay because `readdir` can fail (and in that case we don't want to run the loop, so having the loop inside the `try` seems good on that). But I added another `try catch` inside the loop, so that if we get an IO error for one of the paths it will just ignore that single path, not the entire directory.

Reply:
And now we need a new approve from you because of the push :)